src/common/model.ts

```diff
@@ -58,16 +58,59 @@ export interface UserStat {
   unavailableCount: number;
 }
 
+/**
+ * Represents the place used to get location.
+ * It may be one of three options:
+ * 1. CACHED_ADDRESS - address stored in local storage
+ * 2. BROWSER_GEOLOCATION - address obtained from the browser geolocation API
+ * 3. IP_INFO - address obtained from ipinfo.io API
+ */
 export enum LocationFetchType {
-  IP_INFO,
-  ADDRESS,
-  BROWSER_GEOLOCATION
+  CACHED_ADDRESS, // 'address' in Choo version
+  BROWSER_GEOLOCATION, // 'browserGeolocation' in Choo version
+  IP_INFO // 'ipAddress' in Choo version
 }
```

Comment:
This is not possible to do with a TypeScript `enum`. Like in the Choo app -- where they are represented as string --  it encapsulates where the address comes from: the cache, the browser geolocation or from ipinfo.io.